style.css

```diff
@@ -13,66 +13,110 @@ html {
 }
 
 body {
-   margin: 4rem;
+   padding: 0 4rem;
    font-size: 2rem;
    background-color: #F5FFFA;
-   font-family: 'Loved by the King', cursive;
+   font-family: 'Open Sans', sans-serif;
 }
 
-.heading {
-   padding: 0 2rem 2rem 4rem;
- }
-
 h1 {
    font-family: 'Over the Rainbow';
+   color: #1E90FF;
    float: left;
    clear: right;
-   overflow: hidden;
-	color: #1E90FF;
+   width: 20rem;
+   position: absolute;
```

Comment:
Position absolute will take your element out of the box model. Use with caution. I don't think you should need that here if I remember your site design correctly. Also floats don't do anything on absolutely positioned elements.